contrib/storage-http/src/main/java/org/apache/drill/exec/store/http/HttpXmlOptions.java

```diff
@@ -111,7 +111,7 @@ public String toString() {
   public static class HttpXmlOptionsBuilder {
 
     private int dataLevel;
-    private boolean allTextMode;
+    private Boolean allTextMode;
```

Comment:
I thought there were 3 modes: allTextMode, allNumbersAreDouble mode, and infer-types mode. 

So why is this a boolean vs am enum?

Reply:
@mbeckerle 
In the JSON reader there are two parameters: `allTextMode` and `readAllNumbersAsDouble`.  Both are boolean.    For the XML reader, I chose not to implement the `readAllNumbersAsDouble` parameter because in practice, it requires very clean data.   From using Drill with clients, I can tell you from a lot of personal experience that this was one of the biggest data challenges.   For instance, you'd get data where there was an DOUBLE field and then there would be a row with zero denoted as `0`.   This would then cause schema change exceptions. 

We have actually made significant improvements in Drill's implicit casting rules which do prevent a lot of schema change exceptions and as a result, IMHO, it makes distinguishing between INTs and DOUBLES a lot less important.  So.. out of laziness I decided it wasn't worth it.  I can be convinced otherwise.